peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -77,6 +77,19 @@ <h1 class="py-4">{{ document.title }}</h1>
                         {% endif %}
                       {% endblock %}
 
+                      {% block document-metadata-content-alternative-names %}
+                        {% with document.alternative_names.all as alternative_names %}
+                          {% if alternative_names %}
+                            <dt>{% trans 'Alternative names' %}</dt>
+                            <dd>
```

Comment:
```suggestion
                            <dd class="text-muted">
```

Reply:
We should either do this in CSS for all the metadata, or stop doing it completely. But for now, let's be consistent.